docs/examples/settings_with_custom_parsing.py

```diff
@@ -0,0 +1,24 @@
+# output-json
+import os
+from typing import Any, List
+
+from pydantic import BaseSettings
+
+
+def parse_list(s: str) -> List[int]:
+    return [int(x.strip()) for x in s.split(',')]
+
+
+class Settings(BaseSettings):
+    numbers: List[int]
+
+    class Config:
+        @classmethod
+        def parse_env_var(cls, field_name: str, raw_val: str) -> Any:
+            if field_name == 'numbers':
+                return parse_list(raw_val)
```

Comment:
```suggestion
                return [int(x) for x in s.split(',')]
```

then remove `parse_list` above, I think this is just as easy to read and more compact.

I don't think you need the strip since `int('  123  ') == 123`